projects/pgai/pgai/_install/install.py

```diff
@@ -45,6 +50,18 @@ def _get_vector_extension_schema_sql() -> sql_lib.SQL:
     """)
 
 
+def get_ai_extension_version_sql() -> sql_lib.SQL:
+    return sql_lib.SQL("select extversion from pg_extension where extname = 'ai'")
+
+
+def check_extension_version_result(result: Any | None) -> None:
```

Comment:
I think we should rename this to make it clear that it raises:
```suggestion
def raise_on_old_extension_version(result: Any | None) -> None:
```